cssResponsive/css-responsive_responsive-website/css/styles.css

```diff
@@ -30,19 +30,20 @@ the start of the introduction. */
   --regular-text: 1rem;
   --regular-line-height: 1.5rem;
 
-  /* Calculations */
+  /* Calculations    16px=1rem   */
 
-  /* Calculating the header height */
+  /* Calculating the header height 
+          header-height = 12.6rem 202px */
```

Comment:
In this context it's fine to leave but in a professional context you probably wouldn't want to leave these types of comments in there, just fyi :)

Reply:
yeah. i wasnt sure in the beginning if they wanted us to calculate it or not.
and later i forgot about it.